From d9786540a205bec0b381f339497d7c2cce00376b Mon Sep 17 00:00:00 2001
From: Jonathan Wakely <jwakely@redhat.com>
Date: Wed, 1 Feb 2017 02:07:28 +0000
Subject: [PATCH] Add required header for ostream operations

This file uses std::ostream::operator<<(double) without including
<ostream> and so fails to compile using GCC 7.
---
 k3dsdk/measurement.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/k3dsdk/measurement.h b/k3dsdk/measurement.h
index 9edf7aef..67df5aa0 100644
--- a/k3dsdk/measurement.h
+++ b/k3dsdk/measurement.h
@@ -27,6 +27,7 @@
 
 #include <cassert>
 #include <iosfwd>
+#include <ostream>
 #include <map>
 #include <string>
 #include <typeinfo>
